docs/reference/search/search-your-data/search-across-clusters.asciidoc

```diff
@@ -286,6 +286,321 @@ means the document came from the local cluster.
 <2> This document came from `cluster_one`.
 <3> This document came from `cluster_two`.
 
+
+
+
+
+[discrete]
+[[ccs-async-search-minimize-roundtrips-true]]
+=== Using async search for CCS with ccs_minimize_roundtrips=true
+
+The following <<async-search,async search>> API request searches the `my-index-000001` index on
+three clusters:
```

Comment:
```suggestion
Remote clusters can be queried asynchronously using the 
<<async-search,async search>> API. Async searches accept a
<<ccs-minimize-roundtrips,`ccs_minimize_roundtrips`>> parameter that defaults to
`true`. 

The following request searches the `my-index-000001` index on three clusters 
asynchronously:
```

Reply:
I think it's nice to introduce the Async API and the `ccs_minimize_roundtrips` parameter before jumping into the example.